registration/forms.py

```diff
@@ -1,7 +1,30 @@
-from django.forms import ModelForm
+from django import forms
+from django.contrib.auth.models import User
+from django.forms.models import inlineformset_factory
+from django.forms.extras.widgets import SelectDateWidget
+from django.contrib.auth.forms import UserCreationForm
 from scout.models import Scout
+from scoutmaster.models import Scoutmaster
 
-class ScoutForm(ModelForm):
+ScoutFormSet = inlineformset_factory(User, Scout, can_delete=False, widgets={'dob': SelectDateWidget()})
+ScoutmasterFormSet = inlineformset_factory(User, Scoutmaster, can_delete=False)
+
+class MbuUserCreationForm(UserCreationForm):
```

Comment:
This form captures the minimum we need for you to be considered a "user" of our system.  Check out our models that have a foreign key to user -- those are the different user types.